### DISCUSSION TITLE
How to use microsoft guidance in .net C# or in java

### DISCUSSION THREAD

**Discussion by VadymIgnatiev** [guidance-ai/guidance#427] [Q&A]
Is it possible to connect or use Microsoft guidance from C# .net or Java? Is there an API for these technologies? And if not, is it possible to connect existing API to C# or Java?

**Comment by marcotcr** [comment]
Sorry, there is no API or bindings for other languages yet, nor do we have plans of porting guidance. We do plan on implementing a server version, but I imagine you will still want the client side to be python, since everything in this new release is so pythonic.

P.S. It's not **Microsoft** guidance anymore... just guidance :)
<img src="https://i.imgflip.com/86hfin.jpg" title="made at imgflip.com" style="width: 100px" />

**Reply by VadymIgnatiev** [reply]
Oh, thanks
